app/Data/Articles/FeaturedCollectionData.php

```diff
@@ -16,6 +16,7 @@ class FeaturedCollectionData extends Data
 {
     public function __construct(
         public string $name,
+        public string $slug,
```

Comment:
where is this used for featured collections? they don't link to anything 🤔

Reply:
Yes, but we will need that here: 

> On the right we display the collection images of the featured collections in the article. On hover we display the collection name and on click it navigates to the collection on Dashbrd. On the left side we show a basic count of the amount of featured collections in the article. Similar to the gallery component that already exists

![image](https://github.com/ArdentHQ/dashbrd/assets/132887516/a8bdac54-e843-479b-a9fc-2777d4ead1ba)